processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -163,7 +164,7 @@ public Object get()
       }
 
       @Override
-      public DimensionSelector makeDimensionSelector(final String dimension)
+      public DimensionSelector makeDimensionSelector(final String dimension, final ExtractionFn extractionFn)
```

Comment:
Same question as before, do you have some benchmarks to check the performance impact of these changes?